hoomd/hpmc/integrate.py

```diff
@@ -254,6 +255,9 @@ def test_overlap(self,
 
         For two-dimensional shapes, pass the third dimension of **rij** as zero.
 
+        Attention:
+            `test_overlap` is not yet implemented in HOOMD v3.x.
```

Comment:
We should probably comment the code out or just use `return` to prevent the code from running if a user uses the method.

Reply:
I added a `NotImplementedError` exception.